Dockerfile

```diff
@@ -11,4 +13,4 @@ ENV ELASTIC_LOG_PASSWORD='<ELASTIC-LOG-PASSWORD>'
 RUN apk update && apk add --no-cache bash
 RUN ls -ltra
 COPY gerrit ./
-CMD ./gerrit --gerrit-url=${GERRIT_URL} --gerrit-project=${GERRIT_PROJECT} --gerrit-es-url=${ES_URL}
+CMD ./gerrit --gerrit-url=${GERRIT_URL} --gerrit-project=${GERRIT_PROJECT} --gerrit-es-url=${ES_URL} --gerrit-user=${GERRIT_USERNAME} --gerrit-ssh-key-path=${GERRIT_SSH_KEY}
```

Comment:
Are you sure what to decrypt/encrypt SSH key *path* not the key contents? Just to confirm...

Reply:
The SSHKey is now been sent along with the task running request. Do you think it will be better to just have another flag to handle it and they check the key path if the ssh key is not in the initial request?

Reply:
I only meay you added a flag to pass ssh key location `--gerrit-ssh-key-path=/path/to/key` instead of passing key contents `--gerrit-ssh-key=contents`.

Reply:
Oh got it. I have updated it.